src/core/Akka.API.Tests/LogFormatSpec.cs

```diff
@@ -0,0 +1,112 @@
+// -----------------------------------------------------------------------
+//  <copyright file="LogFormatSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2024 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2024 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Akka.Event;
+using VerifyXunit;
+using Xunit;
+
+namespace Akka.API.Tests;
+
+/// <summary>
+/// Regression test for https://github.com/akkadotnet/akka.net/issues/7255
+///
+/// Need to assert that the default log format is still working as expected.
+/// </summary>
+public sealed class DefaultLogFormatSpec : TestKit.Xunit2.TestKit
+{
+    public DefaultLogFormatSpec() : base("akka.loglevel = DEBUG")
+    {
+    }
+
+    public class OutputRedirector : IDisposable
+    {
+        private readonly TextWriter _originalOutput;
+        private readonly StreamWriter _writer;
+
+        public OutputRedirector(string filePath)
+        {
+            _originalOutput = Console.Out;
+            _writer = new StreamWriter(filePath) { AutoFlush = true };
+            Console.SetOut(_writer);
+        }
+
+        public void Dispose()
+        {
+            Console.SetOut(_originalOutput);
+            _writer.Dispose();
+        }
+    }
+
+    [Fact]
+    public async Task ShouldUseDefaultLogFormat()
+    {
+        // arrange
+        var filePath = Path.GetTempFileName();
+        var probe = CreateTestProbe();
+        Sys.EventStream.Subscribe(probe.Ref, typeof(LogEvent));
+
+        // act
+        using (new OutputRedirector(filePath))
```

Comment:
The error in #7255 was caused not by the formatter on the message changing, but by the fact that we started calling `.ToString` on the wrong thing in v1.5.21. This test is designed to capture STDOUT and verify against that, so either type of mistake will be caught in the future should it occur again.